### ISSUE TITLE
Snapshotting doesn't verify the root.json manifest.

### ISSUE THREAD

**Issue by asraa** [theupdateframework/go-tuf#292]
Currently `Snapshot` will verify signatures on all snapshotted manifests, but does not require verifying the root.json because root is no longer included in the Snapshot. (see https://github.com/theupdateframework/go-tuf/issues/203)

Root.json was removed from snapshot.json here: https://github.com/theupdateframework/taps/blob/master/tap5.md#security-analysis

Should repo managers verify that root.json is valid before snapshotting? As a matter of robustness? Or should we handle this in our own client.

See https://github.com/sigstore/root-signing/issues/238

**Comment by JustinCappos** [comment]
I would think it should verify this.  I'm not sure how you can check that
the snapshot key is current / valid otherwise.

**Comment by joshuagl** [comment]
Agreed that repository tooling should be verifying metadata as the tools proceeds through repository operations.

**Comment by asraa** [comment]
Awesome, the PR opened should do so!
